web/docs/tql2/operators/chart_area.md

```diff
@@ -0,0 +1,56 @@
+# chart_area
+
+Chart values on a X-Y area chart.
+
+```tql
+chart_area x=any, y=any, [x_min=any, x_max=any, resolution=duration, x_log=bool, y_log=bool, group=any, position=string]
+```
+
+## Description
+
+Chart values on X-Y area chart.
+
+### `x = any`
+
+The values to plot on the X-axis.
+
+### `y = any`
+
+The values to plot on the Y-axes. The values are grouped for each unique `x`
+with the given aggregation if specified.
+
+If specified as a record, the field names are used to label the Y-axes.
+
+### `x_min = any (optional)`
+
+If specified, only charts events where `x > from`.
+
+### `x_max = any (optional)`
+
+If specified, only charts events where `x < to`.
```

Comment:
Should this be `>=` and `<=` instead?

Reply:
The operator creates the expression using `gt` and `lt`, so it's reflected here as is. Do you think we change this?

Reply:
My intuition says that this should be a half-open interval $`[min, max)`$.

Reply:
Normally I would be all for half-open intervals, but in this case I think it should be `[min, max]`. You can interpret `x_min` as `x_max` as only having a visual effect (just like `y_min` and `y_max` will), which implies `[min, max]` as you can't visually limit the plot to an open interval.